workers/unity/Packages/com.improbable.gdk.tools/DevAuthTokenUtils.cs

```diff
@@ -9,16 +9,26 @@ namespace Improbable.Gdk.Tools
 {
     public static class DevAuthTokenUtils
     {
+        public static string DevAuthToken => PlayerPrefs.GetString(DevAuthTokenKey);
+
+        private static string DevAuthTokenAssetPath =>
+            Path.Combine("Assets", GdkToolsConfiguration.GetOrCreateInstance().DevAuthTokenDir, "DevAuthToken.txt");
```

Comment:
can also use `Application.dataPath` instead of hardcoding it to "Assets"

Reply:
`AssetDatabase` requires we use relative paths - so I had to hardcode the "Assets"